### PR TITLE
Milestone 4 - Add final require functionalities in app.rb file

### PR DESCRIPTION
**Pull request by mukhammadrizooff**
# In this Project I implement:
- I watched the video with the UI example.
    - [This video](https://www.youtube.com/watch?v=vkkgrhD6aXQ)
- My console app behaves in the same way as in the example.
- I create an `app.rb the file that serves as my console app entry-point. It has methods that I do the following:
  - List all books.
  - List all people.
  - I create a person (teacher or student, not a plain `Person`).
  - Create a book.
  - Create a rental.
  - List all rentals for a given person id.

### CODE REVIEW

File: app.rb [hunk 1]
```diff
@@ -0,0 +1,179 @@
...
+        puts %(Date: #{rental.date}, Book \"#{rental.book.title}\" by #{rental.book.author})
+      end
+    end
+  end
+end
+
+def main
+  app = App.new
+  app.run
+end
```
Comment (er-contreras) [inline app.rb]: - [x] The method main should be in a separate file as main.rb. Follow [this](https://github.com/microverseinc/curriculum-ruby/blob/main/oop/lessons/console_apps.md#console-app-entry-point) explanation to have a hint on how to do it.

**Review by er-contreras — CHANGES_REQUESTED** [review]
Hi @mukhammadrizooff, 

Good job so far!
There are some issues that you still need to work on to prepare your project for the final evaluation but you are almost there!

<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

almost there

## Suggested changes

- [x] It would be great if you can give some instructions in your readme file on how you run the program. 😉
_Check the comments under the review._

You can use as many of my suggestions as you want. If there is anything you would like to skip - feel free to do that. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

File: app.rb (hunk 1 above)
Comment (mukhammadrizooff) [reply app.rb]: ###  @eri8-9  Thank you very much, my friend, for your attention. I will implement all requirement changes

**Review by DeliceLydia — APPROVED** [review]
## Hi @mukhammadrizooff, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
# Congratulations! 🎉 



### Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**